yarn/pom.xml

```diff
@@ -41,28 +42,34 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-yarn-api</artifactId>
+      <version>${hadoop.version}</version>
```

Comment:
This should be `yarn.version` if anything, but should not be necessary. The version is managed in the parent POM. This is not the right way to do this.